industrial_ci/src/env.sh

```diff
@@ -56,22 +56,44 @@ if [ ! "$ROS_REPOSITORY_PATH" ]; then
     esac
 fi
 
-export UBUNTU_OS_CODE_NAME
-if [ -z "$UBUNTU_OS_CODE_NAME" ]; then
-    case "$ROS_DISTRO" in
-    "hydro")
-        UBUNTU_OS_CODE_NAME="precise"
-        ;;
-    "indigo"|"jade")
-        UBUNTU_OS_CODE_NAME="trusty"
-        ;;
-    "kinetic"|"lunar")
-        UBUNTU_OS_CODE_NAME="xenial"
-        ;;
-    *)
-        error "ROS distro '$ROS_DISTRO' is not supported"
-        ;;
-esac
+export OS_CODE_NAME
+export OS_NAME
+export DOCKER_BASE_IMAGE
+
+if [ -n "$OS_NAME" ] && [ -z "$OS_CODE_NAME" ]; then
+    error "please specify OS_CODE_NAME"
 fi
 
+if [ -n "$UBUNTU_OS_CODE_NAME" ]; then
```

Comment:
I think this `if` block exists for backward compatibility only for now-obsolete `UBUNTU_OS_CODE_NAME`. If so, I'd add inline comment to state so, so that in the future it won't get removed by accident.